bricks/template/test.cc

```diff
@@ -274,3 +276,89 @@ TEST(TemplateMetaprogrammingInternalTest, NonemptyConstructorForCombiner) {
   EXPECT_EQ("foo", joined("test"));
   EXPECT_EQ(42, joined(0));
 }
+
+// ********************************************************************************
+// * `current::decay_t` test, moved here from `decay.h`.
+// ********************************************************************************
+template <typename A, typename B>
+inline constexpr bool is_same_v = std::is_same_v<A, B>;
+
+using namespace current;
```

Comment:
Maybe put this part into some `namespace test_current_decay`? The seemingly-global `using namespace current` looks scary, even in the test (keep in mind we have the top-level `make test` target).